src/test/java/com/virtuslab/archunit/ForbiddenMethodsTestSuite.java

```diff
@@ -32,6 +33,14 @@ public void no_classes_should_call_Collections_nCopies() {
         .because("it is confusing as it does not copy the objects but just copies the reference N times")
         .check(importedClasses);
   }
+  @Test
+  public void no_classes_should_call_PropertiesComponent_getInstance_without_args() {
```

Comment:
the test

Reply:
Nit: pls keep abc order of tests + newline before `@Test`